src/Shell/Task/SimpleBakeTask.php

```diff
@@ -109,7 +109,7 @@ public function bake($name)
      * Generate a test case.
      *
      * @param string $className The class to bake a test for.
-     * @return void
+     * @return string
```

Comment:
Which makes the case where the ``no-test`` param is true, the method should return ``false`` as well (line 117 below)